### PR TITLE
feat(FX-3864): Replace request for marketingHubCollections with marketingCollections + list of slugs

### PR DESCRIPTION
**Pull request by nickskalkin**
The type of this PR is: Feature

Resolves [FX-3864]: https://artsyproduct.atlassian.net/browse/FX-3864

### Description

This is a part of [Kaws deprecation](https://www.notion.so/artsy/Deprecating-Kaws-7be8b8b764534ea7838224f06252b99a) story. We want to get rid of marketingHubCollections query to simplify migration from Kaws to Gravity. `marketingHubCollections` query is [literally](https://github.com/artsy/kaws/blob/main/src/Resolvers/Collections.ts#L88) the same as just querying `marketingCollections` with some hardcoded slugs.

This shouldn't change any UI.

Screenshot 2022-04-06 at 16 30 59

I also removed some unused (hopefully) code, discussion is [here](https://artsy.slack.com/archives/C9SATFLUU/p1649250707473079). @damassi Matt suggested to tag you for review since you might know if I am doing something wrong here

[FX-3864]: https://artsyproduct.atlassian.net/browse/FX-3864?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/v2/Apps/Collect/collectRoutes.tsx [hunk 1]
```diff
@@ -146,8 +146,17 @@ function getArtworkFilterQuery() {
       $aggregations: [ArtworkAggregation]
       $shouldFetchCounts: Boolean!
     ) {
-      marketingHubCollections {
-        ...Collect_marketingHubCollections
+      marketingCollections(
+        slugs: [
```
Comment (nickskalkin) [inline src/v2/Apps/Collect/collectRoutes.tsx:150]: slugs are taken from [here](https://github.com/artsy/kaws/blob/main/src/Resolvers/Collections.ts#L89)

**Review by dimatretyak — APPROVED** [review]
Nice PR and changes (especially thanks for removing unused code)

File: src/v2/Apps/Collect/Routes/Collect/index.tsx [hunk 2]
```diff
@@ -46,11 +46,11 @@ export const CollectApp: React.FC<CollectAppProps> = ({
 
   let canonicalHref
   if (medium) {
-    canonicalHref = `${sd.APP_URL}/collect/${medium}`
+    canonicalHref = `${getENV("APP_URL")}/collect/${medium}`
```
Comment (damassi) [inline src/v2/Apps/Collect/Routes/Collect/index.tsx:49]: thank you for updating!

**Review by mdole — COMMENTED** [review]
great work! let's 🚢

**Review by mdole — APPROVED** [review]

**Pull request merged**